src/input.ts

```diff
@@ -7,10 +7,12 @@ import { input } from '@actions-rs/core';
 // Parsed action input
 export interface Input {
     token: string;
+    ignore: string[];
 }
 
 export function get(): Input {
     return {
         token: input.getInput('token', { required: true }),
+        ignore: input.getInput('ignore', { required: false }).split(/, */g),
```

Comment:
It might be better to use `input.getInputList` here

https://github.com/actions-rs/core/blob/master/src/input.ts#L33-L43